boards/arm/stm32mp157c_dk2/Kconfig.defconfig

```diff
@@ -25,3 +25,10 @@ config UART_7
 endif # SERIAL
 
 endif # BOARD_STM32MP157_Dk2
+
+if IPM
```

Comment:
This should be moved in SoC as I don't think another MP1 board will support other config

Reply:
done